packages/integration-tests/src/entities/User.ts

```diff
@@ -1,7 +1,8 @@
-import { UserCodegen } from "./entities";
-
-import { userConfig as config } from "./entities";
-
+import { UserCodegen, userConfig as config } from "./entities";
+/**
+ * Representing a User of the application. Users are the tied to the authentication system and can log in.
+ *
+ * @author Joist */
```

Comment:
Kinda picky, but could we get the `*/` to line up on the last line, like:

```
/**
 * Foo bar.
 *
 * @author Joist
 */
``` 

Er, in a different file down below, this `*/` was placed on the new line...I guess in the codegen file, it's added a `\n` I guess? Not sure why, but is that something we could do here as well?

Reply:
This seems to be fixed now.